TTHAnalysis/python/tools/higgsRecoTTH.py

```diff
@@ -1,17 +1,21 @@
+from __future__ import division
 from PhysicsTools.NanoAODTools.postprocessing.framework.eventloop import Module 
 from PhysicsTools.NanoAODTools.postprocessing.framework.datamodel import Collection as NanoAODCollection
 from CMGTools.TTHAnalysis.tools.nanoAOD.friendVariableProducerTools import declareOutput, writeOutput
 from CMGTools.TTHAnalysis.treeReAnalyzer import Collection as CMGCollection
-
-
+from CMGTools.TTHAnalysis.treeReAnalyzer import * 
+from CMGTools.TTHAnalysis.tools.genParticleProducer import *
 import ROOT, itertools
+from math import *
 
+#bTagCut = 0.3093 if year==2016 else 0.3033 if year==2017 else 0.2770
 class HiggsRecoTTH(Module):
-    def __init__(self,label="_Recl",cut_BDT_rTT_score = 0.0, cuts_mW_had = (50.,110.), cuts_mH_vis = (90.,130.), btagDeepCSVveto = 0.4941, doSystJEC=True):
+    #def __init__(self,label="_Recl",cut_BDT_rTT_score = 0.0, cuts_mW_had = (50.,110.), cuts_mH_vis = (90.,130.), btagDeepCSVveto = 0.4941, doSystJEC=True): #TODO update the values here
```

Comment:
As we discussed, this does not change the cut, because the cut is rewritten when the instance is created in functionsTTH.py

Reply:
yes